run_page/tcx_to_strava_sync.py

```diff
@@ -57,19 +63,21 @@ def get_to_generate_files(last_time):
     to_upload_time_list, to_upload_dict = get_to_generate_files(last_time)
     index = 1
     for i in to_upload_time_list:
-        tcx_file = to_upload_dict.get(i)
+        tcx_file = to_upload_dict.get(i)["file"]
+        type = to_upload_dict.get(i)["type"].lower()
+        strava_type = GARMIN_STRAVA_TYPE_DICT[type]
         try:
-            upload_file_to_strava(client, tcx_file, "tcx")
+            upload_file_to_strava_with_type(client, tcx_file, "tcx", strava_type)
```

Comment:
How about reverting other changes and adding a `force_to_run = False` parameter here instead?